packages/babel-parser/test/fixtures/experimental/decorators/invalid-decorator-object-pattern/input.js

```diff
@@ -0,0 +1 @@
+var { @foo foo } = foo;
```

Comment:
The current main incorrectly allows decorators in pattern. This is now fixed, by not handling decorators at all in `parseBindingProperty`.